web/README.md

```diff
@@ -2,12 +2,18 @@
 
 This is a small example on how to use whisper-client on a web application.
 
-## Usage 
+## Usage
 
-1. Change the whisper path in the code;
-2. Run the commands below;
+1. Add Hydra to Hosts
 
-```bash
-$ go build
-$ ./web
-```
\ No newline at end of file
+    ```bash
+    sudo echo "127.0.0.1 hydra" >> /etc/hosts
+    ```
+
+2. Change the whisper path in the code;
```

Comment:
@claudiosegala, this is not descriptive enough. Either way, If you have to setup a whisper path, I think this should become an environment variable, and well documented in this file.

Reply:
@abilioesteves I think that I meant to change the "localhost:7070" to the whisper path. That information is inside the code. But this is outdated, I will replace with docker-compose usage.